src/Cake.Common/Tools/GitReleaseManager/Close/GitReleaseManagerMilestoneCloser.cs

```diff
@@ -140,12 +139,14 @@ private ProcessArgumentBuilder GetArguments(string token, string owner, string r
             builder.Append("--token");
             builder.AppendQuoted(token);
```

Comment:
Secret

Reply:
Yes, that is a great catch!  Not sure how/why this wasn't like this already?!?